environment.yml

```diff
@@ -4,19 +4,41 @@ name: myria3d
 channels:
   - conda-forge
   - anaconda
+  - pytorch
+  - comet_ml
+  - pyg
 dependencies:
   - python==3.9.*
   - pip
   # cudatoolkit to specify the cuda driver in the conda env
-  - conda-forge::cudatoolkit=11.3.1 # single equal sign there, not a typo
-  - numba==0.55.1
+  # - conda-forge::cudatoolkit=11.3.1 # single equal sign there, not a typo
```

Comment:
Commenting those would lead to a pytorch install that is not compiled with cuda support. [You would need something like this:](https://pytorch.org/get-started/previous-versions/#linux-and-windows-16)
> conda install pytorch==1.11.0 torchvision==0.12.0 torchaudio==0.11.0 cudatoolkit=11.3 -c pytorch

Reply:
fixed